surrealdb/clients/http.py

```diff
@@ -110,43 +130,123 @@ async def _request(
         return response_obj
 
     async def execute(self, query: str) -> List[Dict[str, Any]]:
+        """Executes a query against the SurrealDB server.
+
+        Parameters
+        ----------
+        query: :class:`str`
+            The query to execute.
+
+        Returns
+        -------
+        :class:`List[Dict[str, Any]]`
+            The results of the query.
+        """
         response = await self._request(method="POST", uri="/sql", data=query)
         return response.result
 
     async def create_all(self, table: str, data: Any) -> List[Dict[str, Any]]:
+        """Creates multiple items in a table.
+
+        Parameters
+        ----------
+        table: :class:`str`
+            The table to create the item in.
+        data: :class:`Any`
+            The data to insert into the table.
+
+        Returns
+        -------
+        :class:`Dict[str, Any]`
+            The items created
+        """
         response = await self._request(
             method="POST",
             uri=f"/key/{table}",
             data=jsonlib.dumps(data),
         )
-
```

Comment:
why are we removing these whitespaces? it separates out the logic and removing them is harder to read